plugins/source_testing.go

```diff
@@ -107,8 +107,15 @@ func validateResources(t *testing.T, resources []*schema.Resource) {
 
 	// Make sure every column has at least one value.
 	for i, hasValue := range columnsWithValues {
-		if !hasValue && !(table.Columns[i].Name == "_cq_parent_id" && table.Parent == nil) {
-			t.Errorf("table: %s column %s has no values", table.Name, table.Columns[i].Name)
+		col := table.Columns[i]
+		switch {
+		case col.IgnoreInTests,
+			hasValue,
+			col.Name == "_cq_parent_id" && table.Parent != nil:
+		// nop
+		default:
+			return fmt.Errorf("table: %s column %s has no values", table.Name, table.Columns[i].Name)
```

Comment:
I don't think it should return an error but rather call `t.Errorf`

Reply:
Also I don't think a switch case is needed here but better just a simple `if`

Reply:
> call `t.Errorf`
It makes impossible creating a negative test

> simple `if`
The `if` clause started to look too overloaded